tests/ord-service/tests/subscription_flow_test.go

```diff
@@ -88,11 +88,7 @@ func TestSelfRegisterFlow(t *testing.T) {
 	}()
 
 	// Self register runtime
-	runtimeInput := graphql.RuntimeRegisterInput{
-		Name:        "selfRegisterRuntime",
-		Description: ptr.String("selfRegisterRuntime-description"),
-		Labels:      graphql.Labels{testConfig.ProviderLabelKey: testConfig.ProviderID, tenantfetcher.RegionKey: testConfig.Region},
-	}
+	runtimeInput := fixRuntimeInput("selfRegisterRuntime", ptr.String("selfRegisterRuntime-description"))
```

Comment:
I think this is unnecessary because the new label that is added is already available from testConfig.ProviderLabelKey and testConfig.ProviderID